doc/classes/AnimationNodeTransition.xml

```diff
@@ -16,43 +16,49 @@
 			<return type="int" />
 			<param index="0" name="caption" type="String" />
 			<description>
+			Returns input index which corresponds to [param caption]. If not found, returns [code]-1[/code].
 			</description>
 		</method>
 		<method name="get_input_caption" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="input" type="int" />
 			<description>
+				Returns name of input at given index. This name shows in editor next to node input. [param input] is zero-based index.
```

Comment:
```suggestion
				Returns name of input at given index. This name shows in editor next to node input.
```
"zero-based index" is common usage in the programming, so I don't know if it should be written here or not. There are many other functions that use similar indexes besides TransitionNode, but I believe that not need to write this for all of them.

Reply:
I was thinking that being explicit is usually in line with Godot philosophy, so I thought having it there makes sense. But I do see value in shorter descriptions as well. It does seem that other pages don't define such ranges as well, so for consistency it's better to leave it out. There's some meta documentation in the default captions as well, they start from `state 0` upwards.